cmd/migration/main.go

```diff
@@ -164,6 +170,103 @@ func main() {
 			}
 		}
 	}
+
+	extCredsSaved := []extCredsMigration{}
+	for _, si := range siSaved {
+		labels := map[string]string{"apbAction": "provision", "apbName": si.Spec.FQName}
+		extcredsJSON, err := etcdDao.GetRaw(fmt.Sprintf("/extracted_credentials/%v", si.ID))
+		if etcdDao.IsNotFoundError(err) {
+			continue
+		}
+		if err != nil {
+			revertExtractedCredentials(extCredsSaved)
+			revertJobState(jsSaved, options.MigrationNamespace)
+			revertBindInstance(biSaved)
+			revertServiceInstance(siSaved)
+			revertCrdSavedSpecs(crdSavedSpecs)
+			panic(fmt.Sprintf("Unable to migrate all extracted credentials - %v", err))
+		}
+
+		ec := apb.ExtractedCredentials{}
+		err = json.Unmarshal([]byte(extcredsJSON), &ec)
+		if err != nil {
+			revertExtractedCredentials(extCredsSaved)
+			revertJobState(jsSaved, options.MigrationNamespace)
+			revertBindInstance(biSaved)
+			revertServiceInstance(siSaved)
+			revertCrdSavedSpecs(crdSavedSpecs)
+			panic(fmt.Sprintf("Unable to migrate all extracted credentials - %v", err))
+		}
+		k8scli, err := clients.Kubernetes()
```

Comment:
this feels out of place, like we should've gotten this early on in the process before we even tried to get any of the extracted credentials. I guess it's okay here, just feels weird.

Reply:
Is this something that you want me to change?